OSDOCS 10645 Modify contributors guide to make it clear that AsciiBinder should be run using the container (#76388)
openshift/openshift-docs: contributing_to_docs/tools_and_setup.adoc

This ensures that you are tracking the remote repository to keep your local
repository in sync with it.

== Install Podman or Docker and run AsciiBinder as a container image

When you have the documentation repository cloned and set up, you are ready to
install the software and tools you will use to create the content. All OpenShift
documentation is created in AsciiDoc, and is processed with https://github.com/redhataccess/ascii_binder[AsciiBinder],
which is an http://asciidoctor.org/[AsciiDoctor]-based docs management system.

> The recommended way to run AsciiBinder is by using its container image.

Reviewer: ```suggestion
The recommended way to run AsciiBinder is on a container image using Podman or Docker.
```

filename as the original `.adoc` file you edited, but with an
`.html` extension.

[IMPORTANT]
====

> Previously, installing AsciiBinder directly on your system was recommended. However, on recent Linux distribution releases, AsciiBinder does not work because of wrong Ruby infrastructure versions.

Reviewer: ```suggestion
Previously, installing AsciiBinder directly on your system was recommended. However, on recent Linux distribution releases, AsciiBinder does not work because of Ruby version incompatibility.
```
